src/gemm_v2.cpp

```diff
@@ -971,7 +971,7 @@ miopenStatus_t CallGemmStridedBatchedSequential(const Handle& handle,
                     rocblas_datatype::rocblas_datatype_i32_r,
                     rocblas_gemm_algo::rocblas_gemm_algo_standard,
                     0,
-                    0);
+                    rocblas_gemm_flags_pack_int8x4);
```

Comment:
Not sure whether this flag also applies to int32 data type, see https://rocblas.readthedocs.io/en/latest/functions.html#rocblas-gemm-ex-batched-strided-batched

Reply:
We don't support int32 yet.